rest/recording/get-recording-xml/get-recording-xml.6.x.py

```diff
@@ -4,8 +4,9 @@
 # Your Account Sid and Auth Token from twilio.com/user/account
 account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 auth_token = "your_auth_token"
-recording_sid="RE557ce644e5ab84fa21cc21112e22c485" 
+recording_sid = "RE557ce644e5ab84fa21cc21112e22c485"
 
 client = Client(account_sid, auth_token)
 
-client.api.v2010.accounts(sid=account_sid).recordings(sid=recording_sid).fetch()
+client.api.v2010.accounts(sid=account_sid).recordings(sid=recording_sid
```

Comment:
I would like to, but I haven't found any option that handles this, actually, the problem is when the line is over 80 chars, but yapf doesn't look like solving it (regarding the indentation)